### ISSUE TITLE
Incorrect URIs

### ISSUE THREAD

**Issue by asonnenschein** [usgin/modelmanager#22]
The usginmodels API relies on correct data being served out of http://schemas.usgin.org/contentmodels.json.

Currently, URIs in that file look like this:

```
"http://schemas.usgin.org/uri-gin/ngds/dataschema/activefault/1.1"
```

When they should look like this:

```
"http://stategeothermaldata.org/uri-gin/aasg/xmlschema/activefault/1.1"
```

[This problem is blocking CKAN from generating valid namespaces for OGC services.](https://github.com/ngds/ckanext-ngds/issues/377#issuecomment-37693863)

**Comment by smrgeoinfo** [comment]
@ccaudill can you look into this-- High priority!

**Comment by ccaudill** [comment]
yep - Adrian and I are trying to figure out how these uris get generated in the first place. We'll have something figured out by next week in advance of the EGI install

**Comment by ccaudill** [comment]
Should be done - I'll test the upload and publish at node to be sure

**Comment by ccaudill** [comment]
@asonnenschein Namespace still showing incorrectly as: http://schemas.usgin.org/uri-gin/ngds/dataschema/boreholetemperature/1.5
Should be: http://stategeothermaldata.org/uri-gin/aasg/xmlschema/boreholetemperature/1.5

This was tested at http://node.usgin.org/dataset/il-borehole-temperatures-test3-tier3

**Comment by asonnenschein** [comment]
[@ccaudill See this message](https://github.com/ngds/ckanext-ngds/issues/377#issuecomment-37767612)
